libguard/guard_interface.cpp

```diff
@@ -75,7 +77,7 @@ const fs::path& getGuardFilePath()
 {
     if (guardFilePath.empty())
     {
-        throw std::runtime_error(
+        throw InvalidEntityPath(
```

Comment:
Why throwing `InvalidEntityPath` exception here? Looks libguard initialization does not happen right?

Reply:
Added InvalidEntry